### ISSUE TITLE
Compression/conversion artefacts when compared to console counterparts 

### ISSUE THREAD

**Issue by themanwiththeplan911** [VFansss/mgs2-v-s-fix#160]
**Describe the bug**
Basically every texture has extreme artefacting on PC likely because of the conversion process from Xbox which already was poorly converted from PS2
https://youtu.be/mH2ZVlOLPNI?t=1842
**Have you read the manual?**

Are you sure you have read these pages?
Yes
A LOT of infos can be found there, and usually it's more than enough to solve the most common problems...

**Screenshots**
![extreme texture compression](https://user-images.githubusercontent.com/75482550/107886076-41d69180-6f06-11eb-8dca-f301b258bec7.png)

**Additional description and extra useful things**

**Comment by SammyCola** [comment]
I have dug through the textures previously and this is how the raw assets are stored. If you can extract the textures from the PS2 version and import them into the game using the Solidus tool, you might be able to have a cleaner version of the textures used, though I cannot recall if the DXT5 conversion will affect the quality of the textures or not once it is imported.

**Comment by themanwiththeplan911** [comment]
> 
> 
> I have dug through the textures previously and this is how the raw assets are stored. If you can extract the textures from the PS2 version and import them into the game using the Solidus tool, you might be able to have a cleaner version of the textures used, though I cannot recall if the DXT5 conversion will affect the quality of the textures or not once it is imported.

I don t believe anyone has successfully dumped the PS2 file format(?) correct me if i m wrong i tried to dump them with the texture dump PR that recently showed up on PCSX2 but no luck they come out completely wrong
